mmv1/third_party/terraform/services/storage/resource_storage_bucket_test.go.tmpl

```diff
@@ -824,7 +824,7 @@ func TestAccStorageBucket_update(t *testing.T) {
 				Check: resource.ComposeTestCheckFunc(
 					testAccCheckStorageBucketExists(
 						t, "google_storage_bucket.bucket", bucketName, &recreated),
-					testAccCheckStorageBucketWasRecreated(&recreated, &bucket),
+					testAccCheckStorageBucketWasUpdated(&recreated, &bucket),
```

Comment:
We already have a lost of bucket update tests. Even the next test step will check that only so what is the need of this step? 
If we don't want to test ForceNew behavior then we can completely remove this step otherwise if we somehow need to check recreation of bucket, we can add step with different bucket name and that will force recreation of bucket. I am not sure if that will help here reducing flaky test failures.

Reply:
removed the first step